src/components/interpolatedColorLegend.ts

```diff
@@ -109,6 +123,30 @@ export module Component {
       }
     }
 
+    /**
+     * Apply a gradient to the InterpolatedColorLegend.
+     *
+     * @param {boolean} isGradient Whether there should be a gradient or not.
+     *
+     * @returns {InterpolatedColorLegend} The calling InterpolatedColorLegend.
+     */
+     public gradient(isGradient: boolean): InterpolatedColorLegend {
+       this._isGradient = isGradient;
+       return this;
+     }
+
+     /**
+      * Expand the InterpolatedColorLegend to the full length/height of the chart.
+      *
+      * @param {boolean} isExpanded Whether the legend is expanded or not.
+      *
+      * @returns {InterpolatedColorLegend} The calling InterpolatedColorLegend.
+      */
+    public expand(isExpanded: boolean): InterpolatedColorLegend {
+      this._isExpanded = isExpanded;
+      return this;
+    }
```

Comment:
Can we have the argument be a number instead of a boolean?  It would be the most configurable answer to set the length of the legend